docs/modules/ROOT/partials/storage-ceph-remove-mon.adoc

```diff
@@ -52,26 +67,31 @@ kubectl --as=cluster-admin -n syn-rook-ceph-operator logs -f \
   deploy/rook-ceph-operator
 ----
 
-. Cordon node to {mon-operation} and delete MON pod
+. Cordon old nodes
```

Comment:
Maybe better

```suggestion
. Cordon node(s) to {mon-operation}
```

And ensure that you have `:mon-operation: replace` in the resize how-to